packages/@react-aria/gridlist/src/useGridList.ts

```diff
@@ -38,7 +38,14 @@ export interface GridListProps<T> extends CollectionBase<T>, MultipleSelection {
   disabledBehavior?: DisabledBehavior
 }
 
-export interface AriaGridListProps<T> extends GridListProps<T>, DOMProps, AriaLabelingProps {}
+export interface AriaGridListProps<T> extends GridListProps<T>, DOMProps, AriaLabelingProps {
+  /**
+   * Whether keyboard navigation to focusable elements within grid list items is
+   * via the left/right arrow keys or the tab key.
+   * @default 'arrow'
+   */
+  navigationBehavior?: 'arrow' | 'tab'
```

Comment:
naming??

Reply:
seems pretty good to me, matches linkBehavior which indicates that the way you interact with an element changes due to this prop

Reply:
I would lean towards `keyboardNavigationBehavior`, although it's longer, it avoids confusion with router navigation-related props.

Reply:
ah that's true, good point